object_tracker/match.py

```diff
@@ -123,29 +125,71 @@ def _calc_new_roi(self, M):
         self.roi = Roi(min_x=min_x, min_y=min_y, w=w, h=h)
         return min_x, min_y, max_x, max_y
 
+    def _run_orb(self, image):
+        """
+        this method runs orb
+        """
+        M = None
```

Comment:
Be consistent - here you check whether M is none, but don't catch exceptions (and your caller should check).
In the _run_optical_flow method you don't check M but do catch exceptions.

Reply:
done